Services/Administration/OJS.Services.Administration.Business/SubmissionTypes/SubmissionTypesBusinessService.cs

```diff
@@ -50,6 +51,26 @@ public SubmissionTypesBusinessService(
     public async Task<List<SubmissionTypesInProblemView>> GetForProblem() =>
         await this.submissionTypesDataService.GetAll().MapCollection<SubmissionTypesInProblemView>().ToListAsync();
 
+    public async Task<List<SubmissionTypeInDocument>> GetForDocument()
+        => await this.submissionTypesDataService.GetAll().MapCollection<SubmissionTypeInDocument>().ToListAsync();
+
+    public async Task<bool> AllExist(IEnumerable<SubmissionTypeInSubmissionDocumentAdministrationModel> submissionTypes)
+    {
+        foreach (var submissionType in submissionTypes)
+        {
+            if (!await this.submissionTypesDataService.ExistsById(submissionType.SubmissionTypeId))
+            {
+                return false;
+            }
+        }
```

Comment:
This could be done with a single db query.

Reply:
Done.